data/json/monsters.json

```diff
@@ -2820,8 +2824,48 @@
         },
         ["GRAB", 4]
     ],
-    "description":"A distorted and swollen human body.  Its jaws have elongated into a crocodile like snout, dripping with foul smelling saliva.",
-    "flags":["SEES", "HEARS", "SMELLS", "STUMBLES", "WARM", "BASHES", "GROUP_BASH", "POISON", "BLEED", "GRABS", "NO_BREATHE", "REVIVES", "BONES", "PUSH_MON"]
+    "description":"A distorted human body with sleek muscles.  Its jaws have elongated into a ursine-like snout, dripping with foul smelling saliva.",
+    "flags":["SEES", "HEARS", "WARM", "BASHES", "GROUP_BASH", "POISON", "BLEED", "NO_BREATHE", "REVIVES", "BONES", "PUSH_MON"]
+    },{
+    "type" : "MONSTER",
+    "id" : "mon_zombie_mancroc",
+    "name": "zombie snapper",
+    "species":"ZOMBIE",
+    "default_faction":"zombie",
+    "symbol":"Z",
+    "color":"ltgreen_cyan",
+    "size":"MEDIUM",
+    "material":"flesh",
+    "diff":16,
+    "aggression":100,
+    "morale":100,
+    "speed":100,
+    "melee_skill":4,
+    "melee_dice":3,
+    "melee_dice_sides":4,
+    "melee_cut":8,
+    "dodge":2,
+    "armor_bash":4,
+    "armor_cut":6,
+    "vision_day":25,
+    "vision_night":5,
+    "death_drops": [ "mon_zombie_swimmer_death_drops", 65 ],
```

Comment:
This one is not correct, causes an error on load.

Test your PRs at least once.